servers/zms/src/main/java/com/yahoo/athenz/zms/ZMSImpl.java

```diff
@@ -4320,6 +4333,10 @@ AthenzDomain virtualHomeDomain(Principal principal, String domainName) {
     boolean assertionMatch(Assertion assertion, String identity, String action, String resource,
             String domain, List<Role> roles, List<String> authenticatedRoles, String trustDomain) {
 
+        // Lowercase action and resource in assertion as it is possible to store them case-sensitive
+        assertion.setResource(assertion.getResource().toLowerCase());
```

Comment:
We should now lower-case assertions from the DB before checking them. This change was also made in ZTSAuthorizer and ZPE (ZpeUpdPolLoader)